src/types/common/types/common.types.ts

```diff
@@ -29,3 +29,29 @@ export type Country = {
   name: string
   iso2: string
 }
+
+export type FormInputValueChange<Value, Fields> = (
+  id: string,
+  field: keyof Fields,
+  value: Value
+) => void
+
+export type FormNonInputValueChange<Value, Fields> = (
+  key: keyof Fields,
+  value: Value
+) => void
+
+export type UseFormEventHandler<Fields, Event> = (
+  key: keyof Fields
+) => (event: Event) => void
+
+type FormValidationHandler<TData, TValue> = (
```

Comment:
If you didn`t use **T** character before generic name in other types, remove it here too

Reply:
solved